<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>564</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QPushButton" name="pb_getRule">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>GET Rule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_setRule">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>490</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>SET Rule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_Rule">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>40</y>
     <width>491</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Rule">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>491</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_getLog">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>GET LOG</string>
   </property>
  </widget>
  <widget class="QListWidget" name="list_Rule2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>40</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>10</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>INBOUND</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>10</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>OUTBOUND</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_Rule2">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>420</y>
     <width>501</width>
     <height>61</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_clear_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_getRule_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>360</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>GET Rule</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_setRule_2">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>490</y>
     <width>101</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>SET Rule</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
